katas/main/src/pokerhands/joaquin/Hand.cs

```diff
@@ -0,0 +1,21 @@
+namespace PokerHand.Joaquin;
+
+public class Hand
+{
+    public List<Card> Cards = new List<Card>();
```

Comment:
Normally, is better practice to encapsulate attributes as `private`, very few exceptions can be public if they have a very good reason behind

Reply:
Ok, I'll keep that in mind. Thanks!